axelrod/tournament.py

```diff
@@ -4,8 +4,8 @@
 import multiprocessing
 
 from .game import Game
-from .result_set import ResultSet
-from .tournament_type import RoundRobin
+from .result_set import ResultSet#, ProbEndResultSet
```

Comment:
Same here. If you want to keep the code just put it in another commit / branch for safe keeping?